spec/unit/shoulda/matchers/active_model/validate_presence_of_matcher_spec.rb

```diff
@@ -52,13 +52,56 @@
     end
   end
 
+  context "an ActiveModel with a validation on an attribute that delegates to a has_many association in another ActiveModel" do
+    before do
+      parent = define_active_model_class('Parent', accessors: [:children]).new
+      @model = define_active_model_class('Example', accessors: [:parent]) do
+        delegate :children, :children=, to: :parent
+        validates_presence_of :children
+      end
+      .new(parent: parent)
+    end
+
+    it 'accepts' do
+      expect(@model).to validate_presence_of(:children)
+    end
+  end
+
+  context "an ActiveModel with a validation on an attribute that delegates to a habtm association in another ActiveModel" do
```

Comment:
Line is too long. [124/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.